cmd/snap/cmd_find.go

```diff
@@ -84,8 +84,22 @@ func getPriceString(prices map[string]float64, suggestedCurrency, status string)
 	return formatPrice(price, currency)
 }
 
+type SectionName string
+
+func (s *SectionName) Complete(match string) []flags.Completion {
+	// TODO cache the result
+	cli := Client()
+	sections, _ := cli.GetSections()
```

Comment:
`if err != nil { return nil }`, more explicit and clear than just ignoring the error silently, which tends to raise eyebrows. Also saves the allocation bellow, but that's an irrelevant saving in this context.